sentry-spring-jakarta/src/main/java/io/sentry/spring/jakarta/webflux/ReactorUtils.java

```diff
@@ -62,23 +65,25 @@ public static <T> Mono<T> withSentryHub(final @NotNull Mono<T> mono, final @NotN
   }
 
   /**
-   * Writes the current Sentry {@link IHub} to the {@link Context} and uses {@link
+   * Writes the current Sentry {@link IScopes} to the {@link Context} and uses {@link
    * io.micrometer.context.ThreadLocalAccessor} to propagate it.
    *
    * <p>This requires - reactor.core.publisher.Hooks#enableAutomaticContextPropagation() to be
    * enabled - having `io.micrometer:context-propagation:1.0.2+` (provided by Spring Boot 3.0.3+) -
    * having `io.projectreactor:reactor-core:3.5.3+` (provided by Spring Boot 3.0.3+)
    */
   @ApiStatus.Experimental
+  @SuppressWarnings("deprecation")
   public static <T> Flux<T> withSentry(final @NotNull Flux<T> flux) {
-    final @NotNull IHub oldHub = Sentry.getCurrentHub();
-    final @NotNull IHub clonedHub = oldHub.clone();
+    final @NotNull IScopes oldHub = Sentry.getCurrentScopes();
```

Comment:
Rename to `oldScopes` and `clonedScopes`/`forkedScopes`?

Reply:
Changed in https://github.com/getsentry/sentry-java/pull/3366